site/css/_links.css

```diff
@@ -0,0 +1,5 @@
+.redUnderlineLink {
+  border-bottom: 2px solid red;
+  color: white;
+  text-decoration: none;
+}
```

Comment:
please add an example like _colors and _breakpoints, I believe they are still considered useful by the team.